modules/Core/includes/endpoints/UserInfoEndpoint.php

```diff
@@ -17,18 +17,14 @@ public function __construct() {
     }
 
     public function execute(Nameless2API $api, User $user): void {
-        $query = 'SELECT nl2_users.id, nl2_users.username, nl2_languages.short_code as `locale`, nl2_users.nickname as displayname, nl2_users.joined as registered_timestamp, nl2_users.last_online as last_online_timestamp, nl2_users.isbanned as banned, nl2_users.active as validated, nl2_users.user_title as user_title FROM nl2_users LEFT JOIN nl2_languages ON nl2_users.language_id = nl2_languages.id';
-
-        // Ensure the user exists
-        $results = $api->getDb()->query($query . ' WHERE nl2_users.id = ?', [(int) $user->data()->id]);
-
-        $return = $results->first();
+        $return = new stdClass();
```

Comment:
Does not user info endpoint now missing alot of values such at displayname, user_title , locale and more

as before it got it all from the sql object directly, but now its a empty object where you need to enter all of them yourself